### DISCUSSION TITLE
ColossalAI offers Meilisearch the possibility of personalized training

### DISCUSSION THREAD

**Discussion by Jengro777** [meilisearch/product#563] [Feedback & Feature Proposal]
Refer to: https://github.com/hpcaitech/ColossalAI

I hope that the personalized search function of Meilisrarch can continue to remain open source, and ColossalAI is recommended here, which may be helpful for the development of personalized search.

**Comment by gmourier** [comment]
Hey @Avey777 👋 

Thank you for sharing tools that can help and inspire us.
